### PR TITLE
i#4134: Add opnd check 

### PR DESCRIPTION
**Pull request by johnfxgalea**
Makes minor changes to docs to better clarify some assumptions made by drbbdup.

Also adds a check to ensure that the runtime case operand provided by the user is of pointer-size.

Issue: #4134

### CODE REVIEW

File: ext/drbbdup/drbbdup.c [hunk 1]
```diff
@@ -1476,7 +1476,19 @@ static bool
...
+
+    return false;
+}
+
+static bool
+drbbdup_check_case_opnd(opnd_t case_opnd)
+{
+    /* As stated in the docs, runtime case operand must be a memory reference
+     * and pointer-sized.
+     * */
```
Comment (derekbruening) [inline ext/drbbdup/drbbdup.c:1490]: style nit: extra `*`

File: ext/drbbdup/drbbdup.h [hunk 2]
```diff
@@ -279,9 +281,9 @@ typedef struct {
      */
     void *user_data;
     /**
-     * The maximum number of cases, excluding the default case, that can be associated
-     * with a basic block. Once the limit is reached and an unhandled case is encountered,
-     * control is directed to the default case.
+     * The maximum number of alternative cases, excluding the default case, that can be
```
Comment (derekbruening) [inline ext/drbbdup/drbbdup.h:284]: optional: Rename the field to `non_default_case_limit`?

Comment (johnfxgalea) [reply ext/drbbdup/drbbdup.h:285]: Done.

File: ext/drbbdup/drbbdup.c (hunk 1 above)
Comment (johnfxgalea) [reply ext/drbbdup/drbbdup.c]: Done

**Pull request merged**